app/_src/quickstart/universal.md

```diff
@@ -46,8 +46,8 @@ To explore traffic metrics with the demo app, you also need to [set up Prometheu
 Create a token for Redis and a token for the app:
 
 ```sh
-kumactl generate dataplane-token --name=redis > kuma-token-redis
-kumactl generate dataplane-token --name=app > kuma-token-app
+kumactl generate dataplane-token --tag kuma.io/service=redis --valid-for=30d > kuma-token-redis
+kumactl generate dataplane-token --tag kuma.io/service=app --valid-for=30d > kuma-token-app
```

Comment:
using `--name` and then providing DPP as a YAML to kuma-dp is not really secure because you can impersonate for any service. What you want is to generate token for a specific service. Also valid-for is now required.